client/src/views/Dashboard/Dashboard.js

```diff
@@ -35,14 +42,35 @@ const Dashboard = props => {
     'Due Date',
     'Status'
   ]
+
+  const SelfReviewListingHeader = [
+    'Projects',
+    'From date',
+    'To date',
+    'Due Fom Date',
+    'Status'
+  ]
   const peerReviews = useSelector(
     state => state.peerReviewReducer.userPeerReview
```

Comment:
selector functions as mentioned on chat

Reply:
@RakshandaWankhede  will update in PR #9

Reply:
Done